lib/Alien/Build/Plugin/Build/Autoconf.pm

```diff
@@ -234,6 +238,67 @@ Some reasonable default flags will be provided.
   $self;
 }
 
+sub _pkgconf_wrapper {
+  my ($self, $build) = @_;
+
+  return if !_win;
+
+  my $pk = File::Which::which ($ENV{PKG_CONFIG})
+        || File::Which::which ('ppkg-config')
+        || File::Which::which ('pkg-config.pl')
+        || File::Which::which ('pkg-config');
+
+  if (!defined $pk) {
+    $build->log ("Could not locate ppkg-config or pkg-config in your path:\n");
+    return;
+  }
+  
+  
+  if ($pk =~ /\.bat$/i) {
+    $pk =~ s/\.bat$//i;
+    #  should not die?
+    die "bat file does not have adjacent pkg-config"
+      if !-e $pk;
+  }
+  
+  #  flaky means of determining we are using a perl pkg-config
+  my $data = Path::Tiny->new($pk)->slurp;
```

Comment:
Possibly reliable way without having to slurp the file would be to check if `pkg-config.bat` and `pkg-config.pl` are in the same directory found by `File::Which`.

Reply:
My strawberry perls have pkg-config and pkg-config.bat.  I borrowed the .pl check from A::B::Plugin::PkgConfig::PP.  That might work well enough, though.

Reply:
Ah your right memory has failed me.

Reply:
Done in 8afac421b87c70fb291aea9917aace7f6fb9655f
Let me know if it could be modified.